### PR TITLE
Release v0.7.1 preparation

### PR DESCRIPTION
**Pull request by geriom**
- Contributes to #1568

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ For a list of issues targeted for the next release, see the [23Q2][] milestone.
 
 **Other changes**:
 
-## [0.7.1][releases] - next planned release (unpublished yet)
+## [0.7.1][] - next planned release (unpublished yet)
```
Comment (chalin) [inline CHANGELOG.md:19]:
```suggestion
## [0.7.1][]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ For a list of issues targeted for the next release, see the [23Q2][] milestone.
...
```
Comment (chalin) [inline CHANGELOG.md:20]: Also, drop these lines:

    **Breaking changes**: none yet.
    
    **Other changes**:

**Review by chalin — COMMENTED** [review]
See inline comments. Otherwise LGTM

**Comment by geriom** [comment]
> Is this meant to close #1568?

Just linking to the issue. Not closing it until the full release is published.

**Review by chalin — APPROVED** [review]

**Pull request merged**